spec/fixtures/exampleRelationalResponses.js

```diff
@@ -1,7 +1,7 @@
 export const exampleRelatedToManyResponse = JSON.stringify({
   data: {
     id: '1',
-    type: 'organizations',
+    type: 'todos',
```

Comment:
Unclear why we changed this to organizations in the spec and the mocks. Didn't seem to make any different in the specs and was confusing having stuff like this.

```JavaScript
const todo = store.add('organizations', { id: 1, title: 'Buy Milk' }) 
const todo = new Organization()
```

Reply:
This was when i was testing organization relations, it should've been removed, this ended up being a backend issue with pluralizations.